### PR TITLE
Updated forgot password to work with urls and the Sendgrid templates.…

### PR DESCRIPTION
**Pull request by agundy**
… Also fixed token login

### CODE REVIEW

**Comment by agundy** [comment]
fixes #170

**Comment by kmcnellis** [comment]
Not sending the correct error to the user when there is a error on the server

**Comment by agundy** [comment]
Fixed error handling

File: client/app/account/forgot-password/forgot-password.controller.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-    };
-
-  });
+    if(form.$valid) {
+      Auth.resetPassword($scope.user.email, function(err, res){
+        if (!err){
+          $scope.success = true;
+        }
+        else {
+          notify({ message: "Error: Problem Sending Password Reset Email", classes: ["alert-danger"] });
```
Comment (kmcnellis) [inline client/app/account/forgot-password/forgot-password.controller.js]: Shouldn't be start with caps for every word

**Comment by agundy** [comment]
updated error message

**Pull request merged**